start_menu.h

```diff
@@ -0,0 +1,109 @@
+
+        while (choice != 1 ) {
+        std::cout << "Please select an option:" << std::endl;
+        std::cout << "1. Start Game" << std::endl;
+        std::cout << "2. Settings" << std::endl;
+        std::cout << "3. Quit" << std::endl;
+        
+        int choice;
+        std::cin >> choice;
+
+        switch (choice)
+        {
+        case 1:
+        {
+            // code for starting the game
+            std::cout << "Starting the game..." << std::endl;
+            return 0;
+        }
+        case 2:
+        {
+            // code for settings
+            std::cout << "Opening settings..." << std::endl;
+            std::cout << "chouse a setting you want to change" << std::endl;
+            std::cout << "Please select an option:" << std::endl;
+            std::cout << "1. Text Size" << std::endl;
+            std::cout << "2. Font Weight" << std::endl;
+            std::cout << "3. Face Name" << std::endl;
+            int Setting_shoice;
+            std::cin >> Setting_shoice;
+            switch (Setting_shoice)
+            {
+            case 1:
+            {
+                std::cout << "Pick Number" << std::endl;
+                std::cin >> utils::textsize;
+                utils::customize_text(utils::textsize);
+                
+                break;
+            }
+            case 2:
+            {
+                std::cout << "Pick Weight" << std::endl;
+                std::cout << "1. light" << std::endl;
+                std::cout << "2. SemiLight" << std::endl;
+                std::cout << "3. Normal" << std::endl;
+                std::cout << "4. SemiBold" << std::endl;
+                std::cout << "5. Bold" << std::endl;
+                std::cin >> utils::weightsize;
+                if (utils::weightsize == 1)
+                {
+                    utils::customize_text(utils::textsize, utils::Light);
+                }
+                else if (utils::weightsize == 2)
+                {
+                    utils::customize_text(utils::textsize, utils::SemiLight);
+                }
+                else if (utils::weightsize == 3)
+                {
+                    utils::customize_text(utils::textsize, utils::Normal);
+                }
+                else if (utils::weightsize == 4)
+                {
+                    utils::customize_text(utils::textsize, utils::SemiBold);
+                }
+                else if (utils::weightsize == 5)
+                {
+                    utils::customize_text(utils::textsize, utils::Bold);
+                }
+                
+                break;
+            }
+            case 3:
+            {
+                std::cout << "no idea what to do..." << std::endl;
+                
+                break;
+            }
+            }
+            break;
+        }
+        case 3:
+        {
+            // code for quitting
+            std::cout << "Quitting the game..." << std::endl;
+            return 1;
```

Comment:
`return 1;` indicates a mistake. we want to exit this function and return to `main()`, where we will `return 0;` as well, and exit the program